app/assets/javascripts/discourse/app/templates/components/date-input.hbs

```diff
@@ -6,4 +6,6 @@
   input=(action "onChangeDate")
 }}
 
-<div class="picker-container"></div>
+{{#unless useGlobalPickerContainer}}
```

Comment:
could you explain why we need this please? I will read more next week, but from a first look I would be very opposed to this globalpickercontainer

Reply:
If this element is present, then the date picker container is added inside the modal and that is a problem because the picker will be bound to the modal. If it is not present, then the picker container is added at the end of the document and that works better. It is the way other parts of Discourse use it (for example, bookmarks).

Reply:
Ok, well anyways hopefully we should move to native date picker in the next years :+1: